package.json

```diff
@@ -1,15 +1,7 @@
 {
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*",
-    "packages/config/*"
-  ],
```

Comment:
not necessary in pnpm workspaces, we have a separate file for this

Reply:
👍